### ISSUE TITLE
Update docs with new figures

### ISSUE THREAD

**Issue by ian-r-rose** [coiled/dask-community#14]
See https://github.com/dask/community/issues/135

We should be getting figures back from the designer on retainer next week with tweaked colors for higher contrast (better accessibility).

But we can start going through the docs even before that and replacing/adding figures as necessary.

**Comment by scharlottej13** [comment]
just opened up a PR https://github.com/dask/dask/pull/8401
